src/simulator/privacy_tracker.py

```diff
@@ -89,7 +112,26 @@ def privacy_consumption(self) -> PrivacyBudget:
         basic composition rule.  This will be expanded in the future
         to support advanced composition (see TODO #1 above).
         """
-        return PrivacyBudget(self._epsilon_sum, self._delta_sum)
+        # Take maximum privacy budget spent across all buckets.
+        max_epsilon = 0
```

Comment:
I am a little conflicted about this.  I am not sure if `privacy_consumption` as it is defined will yield useful information.  My current thinking is that a more useful function would be as follows:

```
def remaining_query_count(epsilon):
  """Returns the number of additional queries, each having budget epsilon, that could be supported."""
```

What are your thoughts?

Reply:
I think this is highly related to the usecase. I'd imagine both can be useful--depending on how the algorithm is interacting with the PrivacyTracker. In any case, we can definitely add remaining_query_count in a follow-up PR.

Reply:
Seems reasonable.  Let's go with this for now.